eng/sendtohelix.proj

```diff
@@ -13,10 +13,12 @@
     <TimeoutInSeconds Condition="'$(TimeoutInSeconds)' == ''">600</TimeoutInSeconds>
     <_timeoutSpan>$([System.TimeSpan]::FromSeconds($(TimeoutInSeconds)))</_timeoutSpan>
     
-    <!-- We need to enable xunit reporter so that it parses test results -->
-    <EnableXunitReporter>true</EnableXunitReporter>
+    <!-- We need to enable xunit reporter so that it parses test results
+         Package testing doesn't run on xunit. -->
+    <EnableXunitReporter Condition="'$(IsPackageTesting)' != 'true'">true</EnableXunitReporter>
     
     <TestArchiveRuntimeFile>$(TestArchiveRuntimeRoot)test-runtime-$(BuildConfiguration).zip</TestArchiveRuntimeFile>
+    <TestArchiveRuntimeFile Condition="'$(IsPackageTesting)' == 'true'">$(TestArchiveRuntimeRoot)packages-testPayload-$(ConfigurationGroup).zip</TestArchiveRuntimeFile>
```

Comment:
Just wondering, why do we need a different name on the payload?

Reply:
I just wanted to make a separation and make it clear it was for package testing. When we package the package testing runtime file, targetGroup would be netcoreapp, when calling sendtohelix.proj, the way it is setup it will be allConfigurations. I can make this name more general and define it in Directory.Build.props.